btcjson/chainsvrcmds.go

```diff
@@ -668,6 +668,15 @@ func NewInvalidateBlockCmd(blockHash string) *InvalidateBlockCmd {
 	}
 }
 
+// GetZmqNotificationsCmd defines the getzmqnotifications JSON-RPC command.
```

Comment:
Refer to my [comment](https://github.com/btcsuite/btcd/pull/1555#discussion_r429565992) on your other PR:

Should we put zmq stuff in separate files (ex: `zmq*.go`), or go with `chain*.go` files?

Reply:
Moved to ZMQ prefixed files. Kept the test cases where they are though, to avoid repeating all the boilerplate